### PR TITLE
Improve README and doc

### PR DESCRIPTION
**Pull request by Jiayuan-Gu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,64 +48,29 @@ cd ManiSkill2 && pip install -e .
...
+A GPU with the Vulkan driver installed is required to enable rendering in ManiSkill2. The rigid-body environments, powered by SAPIEN, are ready to use after installation. Test your installation:
 
 ```bash
 # Run an episode (at most 200 steps) of "PickCube-v0" (a rigid-body environment) with random actions
+# Or specify an environment by "-e ${ENV_ID}"
 python -m mani_skill2.examples.demo_random_action
 ```
 
 Some environments require **downloading assets**. You can download all the assets by `python -m mani_skill2.utils.download_asset all` or download task-specific assets by `python -m mani_skill2.utils.download_asset ${ENV_ID}`.
+Please refer to our [documentation](https://haosulab.github.io/ManiSkill2/concepts/environments.html) for all supported environments, and which environments require downloading assets.
```
Comment (StoneT2000) [inline README.md]: grammar

Please refer to our [documentation](https://haosulab.github.io/ManiSkill2/concepts/environments.html) for details on which environments require asset downloads.

File: README.md [hunk 2]
```diff
@@ -50,64 +48,29 @@ cd ManiSkill2 && pip install -e .
...
-
-If multiple CUDA devices are visible, the environment will give an error. If you
-want to interactively visualize the environment, you need to assign the id of
-the GPU connected to your display (e.g., monitor screen).
+---
 
-All soft-body environments require runtime compilation and cache generation. You
-can run the following to compile and generate cache in advance. **This step is
-required before you run soft body environments in parallel with multiple processes.**
+Besides, we provide a docker image (`haosulab/mani-skill2`) on [Docker Hub](https://hub.docker.com/repository/docker/haosulab/mani-skill2/general) and its corresponding [Dockerfile](./docker/Dockerfile).
```
Comment (StoneT2000) [inline README.md]: grammar / word flow

We further provide a docker image (`haosulab/mani-skill2`) on the [Docker Hub](https://hub.docker.com/repository/docker/haosulab/mani-skill2/general) and its corresponding [Dockerfile](./docker/Dockerfile).

File: README.md [hunk 3]
```diff
@@ -130,75 +93,30 @@ env.close()
...
-
-## Demonstrations
-
-We provide a dataset of expert demonstrations to facilitate learning-from-demonstrations approaches, e.g., [Shen et. al.](https://arxiv.org/pdf/2203.02107.pdf).
-The datasets can be downloaded from [Google Drive](https://drive.google.com/drive/folders/1hVdUNPGCHh0OULPCowBClPYIXSwsx-J9) or via command line as shown below.
-
-For those who cannot access Google Drive, the datasets can be downloaded from [ScienceDB.cn](http://doi.org/10.57760/sciencedb.02239).
+Besides, we provide an [example](https://haosulab.github.io/ManiSkill2/getting_started/quickstart.html#interactive-play) of interactively playing with our environments.
```
Comment (StoneT2000) [inline README.md]: grammar/wordflow:

Moreover, you can follow this [example](https://haosulab.github.io/ManiSkill2/getting_started/quickstart.html#interactive-play) to interactively play with our environments with keyboard controls.

File: docs/source/getting_started/quickstart.md [hunk 4]
```diff
@@ -63,6 +63,33 @@ alt: SAPIEN viewer
 ---
 ```
 
+## Vectorized Environment
+
+We provide an implementation of vectorized environments (for rigid-body environments) powered by the SAPIEN RPC-based render server-client system.
```
Comment (StoneT2000) [inline docs/source/getting_started/quickstart.md]: We provide an implementation of fast vectorized environments (for rigid-body environments) powered by the SAPIEN RPC-based render server-client system. Importantly our vectorized environments parallelizes any visual observations / rendering on the GPU, making parallel interaction of environments with visual observations extremely fast. Creating a GPU parallelized vectorized environment is simple:

**Review by StoneT2000 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Jiayuan-Gu) [reply README.md]: I want to express two things in the original sentence: 1) the document lists environments in ManiSkill2; 2) which environments require asset downloads. Your suggestion ignores the former. Could you help rephrase it?

**Review by StoneT2000 — APPROVED** [review]

**Pull request merged**